.eslintrc.json

```diff
@@ -6,6 +6,13 @@
     "airbnb-base/legacy"
   ],
   "rules": {
+    "max-len": [
+      2,
+      {
+        "code": 100
+      }
+    ],
+    "no-console": 0,
```

Comment:
I would suggest to allow console.error only and change all the current `console.log` calls that stand for an error message to `console.error` .

Legit `console.log` calls should keep the eslint comment.